graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/analysis/directed/HasCycles.kt

```diff
@@ -0,0 +1,58 @@
+package com.anaplan.engineering.azuki.graphs.analysis.directed
+
+import com.anaplan.engineering.azuki.core.runner.AnalysisScenario
+import com.anaplan.engineering.azuki.graphs.dsl.GraphScenario
+import com.anaplan.engineering.azuki.graphs.graphA
+
+class HasCycles: GraphScenario() {
```

Comment:
:no_entry_sign: I'd expect to seem boundary testing here.. empty graph, graph with vertices but no edges, graph with vertex with edge to itself etc..

some of these will be eacs, some should be analysis scenarios